src/run/tutorial.rs

```diff
@@ -305,18 +308,18 @@ impl TutorialLesson {
     }
     pub fn disable_player_actions_for_current_phase(
         &self,
-        players: &mut Query<&mut PlayerComponent>,
+        players: &mut Query<(&mut PlayerComponent, &mut WeaponComponent)>,
     ) {
-        for mut player in players.iter_mut() {
+        for (mut player, mut weapon) in players.iter_mut() {
             match self {
                 Self::Ability { .. } => {
-                    player.disable_main_attacks();
+                    weapon.disable();
                 }
                 Self::Attack { .. } => {
                     player.disable_special_attacks();
                 }
                 Self::Movement { .. } => {
-                    player.disable_main_attacks();
+                    weapon.disable();
```

Comment:
One `disable` is on `player: PlayerComponent` and one `disable` in on `WeaponComponent`